app/code/Magento/Eav/Model/Config.php

```diff
@@ -183,6 +192,7 @@ public function __construct(
         $this->_universalFactory = $universalFactory;
         $this->serializer = $serializer ?: ObjectManager::getInstance()->get(SerializerInterface::class);
         $this->scopeConfig = $scopeConfig ?: ObjectManager::getInstance()->get(ScopeConfigInterface::class);
+        $this->storeManager = $storeManager ?: ObjectManager::getInstance()->get(StoreManagerInterface::class);
         $this->attributesForPreload = $attributesForPreload;
```

Comment:
Let's move it to the last position, the same as an argument in constructor
```suggestion
        $this->attributesForPreload = $attributesForPreload;
        $this->storeManager = $storeManager ?: ObjectManager::getInstance()->get(StoreManagerInterface::class);
```